bridge/engine/VideoForwarderRtxReceiveJob.cpp

```diff
@@ -64,22 +67,19 @@ void VideoForwarderRtxReceiveJob::run()
     {
         logger::error("Failed to unprotect srtp %u, mixer %s",
             "VideoForwarderRtxReceiveJob",
-            _ssrcFeedbackContext.ssrc,
+            _rtxSsrcContext.ssrc,
             _engineMixer.getLoggableId().c_str());
         return;
     }
 
-    _ssrcFeedbackContext.lastUnprotectedExtendedSequenceNumber = _extendedSequenceNumber;
-    Vp8Rewriter::rewriteRtxPacket(*_packet, _mainSsrc);
-
-    if (!_ssrcFeedbackContext.videoMissingPacketsTracker.get())
-    {
-        assert(false);
-        return;
-    }
+    _rtxSsrcContext.lastUnprotectedExtendedSequenceNumber = _extendedSequenceNumber;
+    Vp8Rewriter::rewriteRtxPacket(*_packet,
```

Comment:
rewriter will turn this into vp8 rtp packet including payloadtype.
Please speak up if you think this is wrong.

Reply:
@RicardoMDomingues, as we spoke with Olof, it this will disable RTX forwarding. Instead, all RTX packets would be re-transmitted (forwarded) as non-RTX, "normal" VP8 RTP packets on a "main" ssrc (not on a feedback). That, in theory, should get rid of domino effect when clients needs some packets, which was lost on a barbell link. Instead of forwarded RTX, client will receive normal, but out-of-order RTP. That should be ok for the client, even that formally it's NACK requewst was not served.

Reply:
👍